earth_enterprise/src/fusion/fusionui/AssetBase.h

```diff
@@ -95,6 +96,7 @@ class AssetBase : public QMainWindow {
   virtual void SetHidden(bool on);
   virtual void AboutToShowFileMenu();
   virtual void AboutToHideFileMenu();
+  virtual void SaveAndBuild();
```

Comment:
This may not need to be a virtual, as the Save() and Build() methods already are, and this is a convenience slot to invoke them chained together.

Reply:
I wanted to make sure it passed all of the tests first. I'll go ahead and make this change now.